### ISSUE TITLE
redisOpts.keyPrefix option is broken

### ISSUE THREAD

**Issue by alanclarke** [OptimalBits/bull#422]
```
var Queue = require('bull')
var videoQueue = Queue('video transcoding', 6379, '127.0.0.1', { keyPrefix: 'video' })
videoQueue.add({video: 'http://example.com/video1.mov'})
```

![screen shot 2016-12-08 at 12 30 57](https://cloud.githubusercontent.com/assets/640611/21010130/56359baa-bd42-11e6-84dc-d4fed703b6c1.png)
```

**Comment by alanclarke** [comment]
pr: https://github.com/OptimalBits/bull/pull/417

**Comment by manast** [comment]
The PR has a review that needs to be addressed before we can merge it.

**Comment by manast** [comment]
Fixed in 03834c580d72031fd46d1db8498da10309dee029

**Comment by mallove** [comment]
> ```
> var Queue = require('bull')
> var videoQueue = Queue('video transcoding', 6379, '127.0.0.1', { keyPrefix: 'video' })
> ```

Hello -- is the above 4-argument constructor documented somewhere?  It's the only way I can get `bull` to pass my `tls` params to `ioredis`.
